DFE-Digital/Dfe.Academies.External
# 103694 additional details page

Resolves [#103694>](https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_boards/board/t/Apply%20to%20become/Stories/?workitem=103694)

## Type of change


## Expected behaviour


Further information summary page and the additional details page has been added, along with document upload

## Steps to test this PR

1. Go to the application summary page and click Further Information
2. You should be redirected to the Further Information Summary page
3. Clicking start section/change your answers should take you to the Additional Details page.
4. Clicking Save and Continue should save all the fields to the database and the documents should be in Sharepoint

## Prerequisites
